Vungle/VungleRouter.m

```diff
@@ -463,14 +483,18 @@ - (void)clearDelegateForRequestingBanner
 {
     __weak VungleRouter *weakself = self;
     dispatch_async(dispatch_get_main_queue(), ^{
-        [weakself clearDelegateWithState:BannerRouterDelegateStateRequesting
-                             placementID:nil];
+        [weakself clearBannerDelegateWithState:BannerRouterDelegateStateRequesting];
     });
 }
 
-- (void)clearDelegateForPlacementId:(NSString *)placementId
+- (NSString *)parseEventId:(NSString *)adMarkup
 {
-    [self clearDelegateWithState:BannerRouterDelegateStateUnknown placementID:placementId];
+    if (adMarkup) {
```

Comment:
Might we need to check the adMarkup.length : `if (adMarkup.length)`?  thanks :)

Reply:
Updated.